build/clean.js

```diff
@@ -0,0 +1,12 @@
+import path from 'path';
```

Comment:
The build folder contains modules for each compile step ... most are simple.
Side Note: I prefer `gulp` over `grunt` as my build tooling - mostly because grunt had not got a major upgrade for years (they **very recently** fixed this).